File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1214,97 +1065,33 @@ The following code sample demonstrates how to include the hidden slides while co
 
 //Load the PowerPoint presentation to convert.
 IPresentation pptxDoc = Presentation.Open("Sample.pptx");
- 
 //Enable or disable including the hidden slides option in converter settings.
 PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
 pdfConverterSettings.ShowHiddenSlides = true;
- 
 //Convert the documents by passing the settings as parameter.
 PdfDocument pdfDoc = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings);
- 
 //Save the converted PDF file.
 pdfDoc.Save("Sample.pdf");
- 
 //Close the presentation instance
 pptxDoc.Close();
- 
 //Close the PDF instance
 pdfDoc.Close();
-
 {% endhighlight %}
 
 {% highlight vb.net tabtitle="VB.NET" %}
-
 'Load the PowerPoint presentation to convert.
 Dim pptxDoc As IPresentation  = Presentation.Open("Sample.pptx")
- 
 'Enable or disable including the hidden slides option in converter settings.
 Dim pdfConverterSettings As PresentationToPdfConverterSettings  = new PresentationToPdfConverterSettings()
 pdfConverterSettings.ShowHiddenSlides = true
- 
 'Convert the documents by passing the settings as parameter.
 Dim pdfDoc As PdfDocument  = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings)
- 
 'Save the converted PDF file.
 pdfDoc.Save("Sample.pdf")
- 
 'Close the presentation instance
 pptxDoc.Close()
- 
 'Close the PDF instance
 pdfDoc.Close()
-
-{% endhighlight %}
-
-{% endtabs %}
-
-## Recreate Nested Metafile
```

Comment:
You have removed this code snippets. Kindly add it.

Reply:
@Suriya-Balamurugan -Done